### PR TITLE
Vector dimensions of the matrix space and of the euclidean space

### PR DESCRIPTION
**Pull request by tirix**
- Moved `~frlmlvec` to main set.mm
- More theorems about vector space dimension in my mathbox, including the dimension of the matrix space (NxN), and of the euclidean space (N)

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -467694,6 +467784,45 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       CADGSNHOSRDEFSNPQ $.
   $}
 
+  ${
+    $d X x $.  $d Y x $.
+    lbslsat.v $e |- V = ( Base ` W ) $.
+    lbslsat.n $e |- N = ( LSpan ` W ) $.
+    lbslsat.z $e |- .0. = ( 0g ` W ) $.
+    lbslsat.y $e |- Y = ( W |`s ( N ` { X } ) ) $.
+    $( The basis of an atom.  (Contributed by Thierry Arnoux, 20-May-2023.) $)
```
Comment (savask) [inline set.mm:467793]: I've never seen the word "atom" being used in this setting in the literature. Does it just mean "a line"? I don't feel strongly about this, but maybe you could write more explicitly like "A nonzero vector ` X ` is a basis of a line spanned by ` X `."

**Review by savask — COMMENTED** [review]
Cool stuff! I have purely cosmetic remarks.

One of them is about the usage of the term "atom" to mean 1-dimensional subspaces. I don't think it's a widespread term, but now I see there are some theorems about "atoms" in Norman's mathbox, in particular, he has theorem [lsatlspsn2](https://us.metamath.org/mpeuni/lsatlspsn2.html) which looks very similar to your `lbslsat`. What is the relation between these two?

File: set.mm (hunk 1 above)
Comment (tirix) [reply set.mm:467793]: Yes, I was referring to `LSAtom` as defined in Norm's Mathbox, without wanting to pull that definition out. That's exactly what theorem ~[lsatlspsn2](https://us.metamath.org/mpeuni/lsatlspsn2.html) is. 
"Line" and your proposal is probably better, but I could refer to Norm's definition anyway.

**Comment by tirix** [comment]
> One of them is about the usage of the term "atom" to mean 1-dimensional subspaces. I don't think it's a widespread term, but now I see there are some theorems about "atoms" in Norman's mathbox, in particular, he has theorem [lsatlspsn2](https://us.metamath.org/mpeuni/lsatlspsn2.html) which looks very similar to your `lbslsat`. What is the relation between these two?

Actually `~lsatlspsn2` is purely expanding the definition of "Atoms", I did not find any theorem in Norm's mathbox which would match `lbslsat`.

**Review by avekens — APPROVED** [review]
great!

**Pull request merged**